STL_Extension/doc/STL_Extension/CGAL/Compact_container.h

```diff
@@ -545,7 +545,7 @@ const T& operator[] (size_type i) const;
 /*!
 returns the element at pos `i` in the container.
 
-\pre `is_used(i) == true` and \f$ 0 \leq \f$ `i` \f$ < \f$ `capacity()`
+\pre `\link is_used(size_type)const `is_used(i)`\endlink`== true` and \f$ 0 \leq \f$ `i` \f$ < \f$ `capacity()`
```

Comment:
Looks like an odd number of backticks, probably first one shouldn't be there.

(overnight documentation build https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-6.2-Ic-13/logs_master/STL_Extension.log, rest might be collateral damage).